cmd/cri-containerd/options/options.go

```diff
@@ -18,38 +18,56 @@ package options
 
 import (
 	"flag"
+	"os"
+	"time"
 
+	"github.com/BurntSushi/toml"
 	"github.com/containerd/containerd"
 	"github.com/spf13/pflag"
 )
 
-// CRIContainerdOptions contains cri-containerd command line options.
-type CRIContainerdOptions struct {
+const (
+	// "default path to the config file"
```

Comment:
nit: Why quote here? Golang comment usually start with the thing itself, `configFilePathArgName is the flag name for XXX`.